projects/ui/src/graph/graphql.schema.json

```diff
@@ -44754,7 +44754,7 @@
           },
           {
             "name": "index",
-            "description": "The Pod Index of the listed Plot.",
+            "description": "The absolute index of the listed Plot in the Pod Line.\n\nMeasured from the front, so the Listing contains all Pods between\n(index) and (index + totalAmount).\n\nAn example where the podLine is 50,000 but the index is 150,000:\n   0         the first Pod issued\n   100,000   harvestableIndex\n   150,000   index\n",
```

Comment:
I'm hoping someone can provide some context about why this change happened, presumably after `yarn generate`.

Reply:
yup. if you follow the yarn generate command, you'll see it does a graphql-codegen, which pulls this schema from a hardcoded url. Must've changed upstream

Reply:
OK just making sure it's expected and it's fine to commit. Thanks.